wgpu-core/src/command/bundle.rs

```diff
@@ -712,12 +743,15 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         num_dynamic_offsets,
                         bind_group_id,
                     } => {
+                        let err_ctx = "In a set_bind_group command";
+
                         let max_bind_groups = device.limits.max_bind_groups;
                         if (index as u32) >= max_bind_groups {
                             Err(RenderCommandError::BindGroupIndexOutOfRange {
```

Comment:
can we be smarter in these cases? i.e.
```rust
Err(pass_err(err_ctx)(RenderCommandError::{..}.into()))?
```
Otherwise, it's a bit strange to construct an `Err` only to follow-up with `map_err`...

Reply:
Yes that would be possible, I just did a blanked pass with the map functions for now, but I can clean up those cases.